src/app/_utils/createMetadata.ts

```diff
@@ -43,16 +43,16 @@ export function createMetadata({
 
   return {
     title: overrideDefaultTitle
-      ? { absolute: parsedEnrichedSEO.title }
-      : parsedEnrichedSEO.title,
+      ? { absolute: parsedEnrichedSEO.title as string }
+      : (parsedEnrichedSEO.title as string),
```

Comment:
The reason for the type errors and need for assertion is because we switched to using SeoMetadataSchemaWithOptionalTitle but at this we can just keep using SeoMetadataSchema - We will already have passed the title.